installation/resources/installer-config-local.yaml.tpl

```diff
@@ -35,6 +35,7 @@ data:
   global.isLocalEnv: "true"
   global.domainName: "kyma.local"
   global.adminPassword: ""
+  global.minikubeIP: ""
```

Comment:
probable we need to introduce appropriate changes to kyma-cli https://github.com/kyma-project/cli/blob/3fae2a44236192e7b41867c4554f072bdfc47247/pkg/kyma/cmd/install/cmd.go#L844 to set minikubeIP for `installation-config-overrides`

Reply:
yes, I plan that to do in the same ticket as soon this PR is merged